### ISSUE TITLE
Content should be read in accessibility mode

### ISSUE THREAD

**Issue by Moyck** [ponnamkarthik/FlutterToast#382]
Android will read content in accessibility mode, but not iOS.

**Comment by ponnamkarthik** [comment]
Plugin i am using for toast for ios doesnt suport accessibility

i will try to add the support
